cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactRepositoryTest.java

```diff
@@ -325,13 +335,18 @@ public void testMacroPlugin() throws Exception {
           Plugin pluginInfo = new Plugin(entry.getKey().getArtifactId(), pluginClass,
                                          PluginProperties.builder().add("class.name", TEST_EMPTY_CLASS)
                                            .add("timeout", "10")
-                                           .add("name", "${macro}")
+                                           .add("name", "${name}")
```

Comment:
The MacroParser has some complex tests in MacroParserTest. I will copy one of those tests and put it here.